src/app/shared/components/card/card.component.html

```diff
@@ -0,0 +1,6 @@
+<div class="card">
+    <div class="header p-4">
+        <ng-content select="[cardHeader]"></ng-content>
+    </div>
+    <ng-content></ng-content>
+</div>
```

Comment:
The `div.card` element seems redundant. The purpose it has can be moved to the host element.
Instead of using the header class, the `header` element can be used instead. So this becomes:
```html
<header class="p-4">
    <ng-content select="[cardHeader]"></ng-content>
</header>
<ng-content></ng-content>
```

Reply:
I get some weird results with the border-radius and box-shadow when applied to the host...

Reply:
I made an example that gives the same result, where the card component files looks like:
```html
<!-- card.component.html -->
<header class="p-4">
  <ng-content select="[cardHeader]"></ng-content>
</header>
<ng-content></ng-content>
```
```scss
// card.component.scss
:host {
  display: block;
  background-color: var(--white-color);
  border-radius: 8px;
  box-shadow: 0 0 15px 1px var(--gray-color);

  & > header {
    display: flex;
    justify-content: space-between;
  }
}
```
The result is:
<img width="1814" alt="Skærmbillede 2024-03-20 kl  21 29 04" src="https://github.com/tv2/ng-sofie/assets/11258272/fa27c2af-5773-462f-8b2e-8a4626393ab3">

Reply:
Nice suggestion. It has been updated